STMReco/fcl/plotSTMWaveformDigis.fcl

```diff
@@ -11,8 +11,12 @@ source : {
 }
 
 services : {
-  message : @local::Services.Core.message
+  message : { @table::Services.Core.message }
   TFileService : { fileName : "stmWaveformDigis.root" }
+  GeometryService : { @table::Services.Core.GeometryService }
+  GlobalConstantsService : { @table::Services.Core.GlobalConstantsService }
+  DbService : { @table::Services.Core.DbService }
+  ProditionsService : { @table::Services.Core.ProditionsService }
```

Comment:
I think it would be a bit cleaner and more maintainable to write
```
services : { @table::Services.Core
   TFileService   : { fileName : "stmWaveformDigis.root" }
}
```
or move the TFileService line to dotted format below

Reply:
Done here: [d81efd9](https://github.com/Mu2e/Offline/pull/879/commits/d81efd97f552402209c320194547233db54ce020)